src/services/checkS3BucketData.ts

```diff
@@ -0,0 +1,56 @@
+import { DataRequestParams } from '../types/dataRequestParams'
+import { S3BucketDataLocationResult } from '../types/s3BucketDataLocationResult'
+import { listS3Objects } from './listS3Objects'
+import { getEnv } from '../utils/helpers'
+import { generateS3ObjectPrefixes } from './generateS3ObjectPrefixes'
+
+export const checkS3BucketData = async (
+  dataRequestParams: DataRequestParams
+): Promise<S3BucketDataLocationResult> => {
+  console.log('Looking for S3 data using params', dataRequestParams)
+
+  const prefixes = generateS3ObjectPrefixes(
+    dataRequestParams.dateFrom,
+    dataRequestParams.dateTo
+  )
+
+  //TODO: add handling for when there is no data available for requested dates
```

Comment:
Can't we do this now? Just by looking to see if there's anything there at all? If we're not doing this in this story, not sure where we are doing it?